### PR TITLE
OPSEXP-1787 Add Control Center to Helm deployment

### PR DESCRIPTION
**Pull request by upandey123**

### CODE REVIEW

File: helm/alfresco-content-services/Chart.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@ dependencies:
     alias: postgresql-syncservice
     condition: >-
       alfresco-sync-service.syncservice.enabled,postgresql-syncservice.enabled
+  - name: common
+    alias: alfresco-admin-app
+    repository: https://activiti.github.io/activiti-cloud-helm-charts
+    version: 7.6.0
+    condition: >-
+      alfresco-content-services.alfresco-admin-app.enabled,alfresco-admin-app.enabled
```
Comment (alxgomz) [inline helm/alfresco-content-services/Chart.yaml:39]: First condition doesn't make sense to me (could also be removed from adw imho)

```suggestion
      alfresco-admin-app.enabled
```

File: helm/alfresco-content-services/values.yaml [hunk 2]
```diff
@@ -79,7 +79,7 @@ repository:
       memory: "3000Mi"
   # -- The startup probe to cover the worse case startup time for slow start containers
   startupProbe:
-    periodSeconds: 30
+    periodSeconds: 180
```
Comment (alxgomz) [inline helm/alfresco-content-services/values.yaml:82]: That is probably only valid for your env that is slow. We don't want that as a default for prod env

**Review by alxgomz — COMMENTED** [review]
minor fixes. Otherwise looks ok

File: helm/alfresco-content-services/Chart.yaml (hunk 1 above)
Comment (upandey123) [reply helm/alfresco-content-services/Chart.yaml]: Fixed

File: helm/alfresco-content-services/values.yaml (hunk 2 above)
Comment (upandey123) [reply helm/alfresco-content-services/values.yaml]: Fixed

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/values.yaml [hunk 3]
```diff
@@ -645,6 +645,38 @@ alfresco-digital-workspace:
...
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+  image:
+    repository: quay.io/alfresco/alfresco-admin-app
+    tag: 7.6.0
+    pullPolicy: IfNotPresent
+  env:
+    APP_CONFIG_AUTH_TYPE: "BASIC"
+    API_URL: "{protocol}//{hostname}{:port}"
```
[the remark this answers is not in the document]
Comment (upandey123) [reply helm/alfresco-content-services/values.yaml:670]: Done

File: helm/alfresco-content-services/values.yaml [hunk 4]
```diff
@@ -645,6 +645,39 @@ alfresco-digital-workspace:
...
+  nodeSelector: {}
+  enabled: true
+  nameOverride: "alfresco-cc"
+  service:
+    envType: frontend
+  ingress:
+    path: /control-center
+    annotations:
+      kubernetes.io/ingress.class: nginx
+     # nginx.ingress.kubernetes.io/proxy-body-size: "5g"
```
Comment (alxgomz) [inline helm/alfresco-content-services/values.yaml:658]: please remove this comment.

File: helm/alfresco-content-services/Chart.yaml [hunk 5]
```diff
@@ -31,12 +31,18 @@ dependencies:
...
+  - name: common
+    alias: alfresco-admin-app
+    repository: https://activiti.github.io/activiti-cloud-helm-charts
+    version: 7.6.0
+    condition: >-
+      alfresco-admin-app.enabled
   - name: common
     alias: alfresco-digital-workspace
     repository: https://activiti.github.io/activiti-cloud-helm-charts
     version: 7.3.0
```
Comment (alxgomz) [inline helm/alfresco-content-services/Chart.yaml:43]: that one could also be updated to 7.6.0

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/values.yaml [hunk 6]
```diff
@@ -645,6 +645,39 @@ alfresco-digital-workspace:
...
+  resources:
+    limits:
+      cpu: 1
+      memory: 1024Mi
+    requests:
+      cpu: 150m
+      memory: 256Mi
+
+# -- Choose if you want Office Online Integration capabilities (Alfresco Collaboration Connector for Microsoft 365)
+
```
Comment (gionn) [inline helm/alfresco-content-services/values.yaml:680]:
```suggestion
```

**Review by gionn — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/values.yaml (hunk 4 above)
Comment (upandey123) [reply helm/alfresco-content-services/values.yaml:658]: done

File: helm/alfresco-content-services/Chart.yaml (hunk 5 above)
Comment (upandey123) [reply helm/alfresco-content-services/Chart.yaml:43]: done

File: helm/alfresco-content-services/values.yaml (hunk 6 above)
Comment (upandey123) [reply helm/alfresco-content-services/values.yaml:680]: deleted the duplicate line.

**Comment by upandey123** [comment]
Re-run this on flux terraform alfresco pipeline and able to open Control Center.
https://github.com/Alfresco/terraform-alfresco-pipeline/actions/runs/3531093576/jobs/5923884683

**Comment by upandey123** [comment]
Screenshot 2022-11-23 at 3 55 26 PM

PFA screenshot!

**Review by alxgomz — CHANGES_REQUESTED** [review]
Pre-commit check failed again, please fix them. Make sure you have pre-commit installed on your machine so you can at least run checks before you push to origin.
Regarding the screenshot. Please try loging in as an admin user , not devops.

**Comment by gionn** [comment]
> Pre-commit check failed again, please fix them

just run `pre-commit run --all-files helm-docs` locally and then commit the result.

**Review by gionn — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**